.github/workflows/Cleanup Caches by a branch.yml

```diff
@@ -8,7 +8,11 @@ on:
   pull_request:
     branches: [main, 'ODS-*']
     types: [ closed ]
-  
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
```

Comment:
This workflow executes only when a PR is closed, so there will not be an in progress workflow to cancel. This change can be reverted.

Reply:
Revert it back . Thanks